calendar-kata/pom.xml

```diff
@@ -1,19 +1,11 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Copyright 2021 The Bank of New York Mellon.
-  ~
-  ~ Licensed under the Apache License, Version 2.0 (the "License");
-  ~ you may not use this file except in compliance with the License.
-  ~ You may obtain a copy of the License at
-  ~
-  ~     http://www.apache.org/licenses/LICENSE-2.0
-  ~
-  ~ Unless required by applicable law or agreed to in writing, software
-  ~ distributed under the License is distributed on an "AS IS" BASIS,
-  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  ~ See the License for the specific language governing permissions and
-  ~ limitations under the License.
-  -->
+Copyright © The Bank of New York Mellon 2021
```

Comment:
Missing `All rights reserved`

`Copyright (c) 2021, The Bank of New York Mellon. All rights reserved.`